### DISCUSSION TITLE
Does pandapower have a single phase simulator or is it always three phase balanced?

### DISCUSSION THREAD

**Discussion by kiedanski** [e2nIEE/pandapower#1071] [Q&A]
Hello,

I'm trying to understand how to read the parameters of the different data structures.

I am currently writing the optimization problem of a market with gird constraints and later I am using pandapower to simulate the grid with the values obtained from my optimization problem to get the real voltage deviations, line loadings, line losses, etc.

I am having some trouble with the maximum current bound. My optimization problem is single phase, and I understood that the parameter max_i_ka was for a single phase network. Nevertheless, I did not get good results until I tried to use sqrt(3) * max_i_ka as the bound (in my optimization problem).

Is this the expected transformation from pandapower data to a single phase system?

If that is indeed the case, is the pandapower simulation always a three phase one (albeit balanced)?

Best,

**Comment by lthurner** [comment]
If you are talking about the symmetric power flow (pp.runpp), all values are considered for all three phases. A power value (i.e. net.load.pn_mw) is the summed power over all three phases. The max_i_ka value relates to the sum of the currents in all three phases. Depending on the format in which you have your values, a transformation with sqrt(3) might be necessary.
